src/ts/components/seekbar.ts

```diff
@@ -211,6 +262,14 @@ export class SeekBar extends Component<SeekBarConfig> {
     // update bufferlevel when a segment has been downloaded
     player.on(player.exports.PlayerEvent.SegmentRequestFinished, playbackPositionHandler);
 
+    player.on(player.exports.PlayerEvent.SourceLoaded, () => {
+      if (player.isLive()) {
```

Comment:
This flag is not necessarily known in `SourceLoaded`, and can also change during playback. So you need to additionally use the `LiveStreamDetector`.